extensions/hdfs-storage/src/main/java/io/druid/storage/hdfs/HdfsDataSegmentPusher.java

```diff
@@ -69,6 +71,14 @@ public String getPathForHadoop(String dataSource)
   public DataSegment push(File inDir, DataSegment segment) throws IOException
   {
     final String storageDir = DataSegmentPusherUtil.getHdfsStorageDir(segment);
+
+    log.info(
+        "Uploading segment[%s] to HDFS at location[%s/%s]",
+        segment.getIdentifier(),
+        config.getStorageDirectory(),
+        storageDir
```

Comment:
this should use the outFile path created below to make sure the log message is consistent with what it's doing